test/stat_tracker_test.rb

```diff
@@ -105,13 +161,12 @@ def test_it_knows_tie_percentage
 
     assert_equal 0.20, stat_tracker.percentage_ties
   end
-#
-#---------------LeagueStatisticsTests
 
+#---------------LeagueStatisticsTests
   def test_it_can_count_of_teams
     game_path = './fixture/games_count_teams.csv'
-    team_path = './fixture/teams.csv'
-    game_teams_path = './data/game_teams.csv'
+    team_path = './data/teams.csv'
+    game_teams_path = './fixture/game_teams_empty.csv'
```

Comment:
We may have to keep in mind that we're using an empty dummy here. Depending on mocks and stubs usage (not sure how that works yet), we would have to be mindful that not all methods use the same dummies.